README.md

```diff
@@ -34,11 +34,11 @@ Please edit this README.md file with your answers to the worksheet questions.
 
 ## Question 4
 
-### a
+### a The two if statements are equivalent because they both get the same end result.
+The first one will only run if both files a and b exist 
+The second one will also only run if both files a and b exist
+They differ on how they check, the first one checks is both a and b exist at the same time while the second checks if one or the other doesn't exist.
```

Comment:
This is true, however you were asked to justify this reasoning using boolean logic -- i.e. if `A` and `B` represent the file existence checks, then the two programs are equivalent by the identity in question 3b.